packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -41,13 +42,6 @@ enum VerifiedCredentialState {
   PENDING,
 }
 
-enum ApplicationQuestions {
```

Comment:
question: we don't need to make RM "backward-compatible" with the old application question labels, right?

Reply:
afaik no we dont't